.github/workflows/deny.yml

```diff
@@ -0,0 +1,32 @@
+name: Deny
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: ["main"]
+    paths-ignore: ["*.md", "*.png", "*.svg", "LICENSE-*"]
+  merge_group:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  deny:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        checks:
+          - advisories
+          - bans licenses sources
+
+    # Prevent sudden announcement of a new advisory from failing ci:
+    continue-on-error: ${{ matrix.checks == 'advisories' }}
```

Comment:
Would we catch it, if it continues on error? I am fine failing CI and requiring a whitelisting pull request.

Reply:
I can make the change. This is taken from the recipe on the URL above.

Reply:
No strong opinion. I assume it is easier to loosen this in the future, than manually checking whether the CI step succeeded.